apps/web/src/app/play/checker/_components/global-monitoring.tsx

```diff
@@ -11,6 +11,9 @@ import {
   CardTitle,
 } from "@/components/marketing/card";
 
+import { Globe } from "@/components/marketing/monitor/globe";
```

Comment:
Why do. we import globe ?

Reply:
The Globe component is imported but not used. I am remove it to clean up the codebase.